packages/shorebird_cli/test/src/commands/preview_command_test.dart

```diff
@@ -125,7 +125,10 @@ void main() {
         () => codePushClientWrapper.getApps(),
       ).thenAnswer((_) async => [app]);
       when(
-        () => codePushClientWrapper.getReleases(appId: appId),
+        () => codePushClientWrapper.getReleases(
+          appId: appId,
+          sideloadableOnly: true,
```

Comment:
```suggestion
          sideloadableOnly: any(named: 'sideloadableOnly'),
```

nit but shouldn't this be an any matcher for consistency?

Reply:
Done